lib/img-workers.js

```diff
@@ -0,0 +1,62 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const P = require('./promise');
+
+const AppError = require('./error');
+const config = require('./config');
+const logger = require('./logging')('img_workers');
```

Comment:
Thanks for the rename, I think it reads clearer now.  IIUC this will change the name of some log messages from "workers" to "img" workers, so we should check with @jrgm whether any monitoring on the ops side will need to change to match.  @jrgm?